### ISSUE TITLE
[Improvement]: Community Tools panel should scroll verticaly

### ISSUE THREAD

**Issue by cupangafk** [neon-nyan/Collapse#212]
### Affected Version

1.70.12

### Bug Behavior

this is not a bug (and yet no form to create an "Improvement")

why dont we split this section into 2 section and scroll them verticaly ?
![image](https://github.com/neon-nyan/Collapse/assets/66926348/f17b2d0d-fac5-498c-a067-cf61fc16d2bf)

its not a bug, its fully functional, its just not intuitive from my experience as a user

i know this is not gonna be ended up in your num 1 priority, take your time :)

### Expected Behavior

-

### Steps to reproduce

-

### Additional Information

ah yes, please add Improvement issue form for this kind of issue in the future (from me again maybe)

**Comment by bagusnl** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Vertical Scroll on Community Tools](https://trello.com/c/35jSidN7/4-vertical-scroll-on-community-tools)

**Comment by bagusnl** [comment]
Hi, sorry for the last response for this. First of all, thank you for expressing your idea for Collapse project!

We have discussed with the core devs for this and we have decided to not implement it as the scrolling works fine now and we don't seem to find it as well translated to the implementation. As we have quite a lot of community tools implemented (maybe more in the future), vertical scrolling Community Tools tab will be quite a scrolling to go through and find the items user wants, so with the scrolling in that tab fixed, we think the current implementation is already quite nice as is.

Though, feel free for anyone to chime in about this in the comments and we'll rethink about it.
